website/docs/r/container_registry.html.markdown

```diff
@@ -52,12 +52,20 @@ The following arguments are supported:
 
 * `tags` - (Optional) A mapping of tags to assign to the resource.
 
-* `georeplication_locations` - (Optional) A list of Azure locations where the container registry should be geo-replicated.
+* `georeplication_locations` - (Optional / Deprecated in favor of georeplications) A list of Azure locations where the container registry should be geo-replicated.
```

Comment:
We need to quote the attribute so that the document renderer could render it differently comparing to normal text
```suggestion
* `georeplication_locations` - (Optional / **Deprecated in favor of `georeplications`**) A list of Azure locations where the container registry should be geo-replicated.
```